extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -117,7 +117,6 @@ public function getWikiVariables() {
 			'homepage' => $this->getHomepageUrl(),
 			'id' => (int) $wgCityId,
 			'language' => [
-				'user' => $wgLang->getCode(),
 				'userDir' => SassUtil::isRTL() ? 'rtl' : 'ltr',
```

Comment:
I considered that, but it's not trivial - Mercury's backend is making SCSS ltr / rtl decision on backend side. I can create a ticket for investigating if we can use a different way to act there.